### DISCUSSION TITLE
get_balancer(): balancer not found for nil

### DISCUSSION THREAD

**Discussion by woshitianbz** [Kong/kong#12863] [Help]
Hello, Kong has 10 instances, and only one instance reported an error after changing the target. What could be the reason for this? Thank you.

 [error] 26#0: *12120115240 [lua] balancer.lua:620: get_balancer(): balancer not found for nil, will create it, client: XXXXXXXX, server: kong, request: "POST /xx/xx/xx HTTP/1.1", host: "xxxxxxxxx"
 [error] 26#0: *12120115240 lua entry thread aborted: runtime error: /usr/local/share/lua/5.1/kong/runloop/balancer.lua:405: attempt to index a nil value
stack traceback:
coroutine 0:
	/usr/local/share/lua/5.1/kong/runloop/balancer.lua: in function 'create_balancer_exclusive'
	/usr/local/share/lua/5.1/kong/runloop/balancer.lua:469: in function 'create_balancer'
	/usr/local/share/lua/5.1/kong/runloop/balancer.lua:621: in function 'get_balancer'
	/usr/local/share/lua/5.1/kong/runloop/balancer.lua:883: in function 'execute'
	/usr/local/share/lua/5.1/kong/runloop/handler.lua:823: in function 'balancer_execute'
	/usr/local/share/lua/5.1/kong/runloop/handler.lua:1166: in function 'after'
	/usr/local/share/lua/5.1/kong/init.lua:714: in function 'access'
	access_by_lua(nginx-kong.conf:89):2: in main chunk, client: xxxxxxx, server: kong, request: "POST /xx/xx/xx HTTP/1.1 ", host: "xx/xx/xx"

**Comment by Water-Melon** [comment]
Thanks for your feedback. Could you open an issue with some simple steps and configurations to reproduce this error? That would be very appreciate.
